packs/data/pfs-season-3-bestiary.db/explosive-rats.json

```diff
@@ -0,0 +1,396 @@
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "fVhwJ2D5j02MQI48",
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.bestiary-ability-glossary-srd.rqfnQ5VHT5hxm25r"
+                }
+            },
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "Scent (Imprecise) 30 feet",
+            "sort": 300000,
+            "system": {
+                "actionCategory": {
+                    "value": "interaction"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>@Localize[PF2E.NPC.Abilities.Glossary.Scent]</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [],
+                "slug": "scent",
+                "source": {
+                    "value": "Pathfinder Bestiary"
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": []
+                },
+                "trigger": {
+                    "value": ""
+                },
+                "weapon": {
+                    "value": ""
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "JhtGLa6CH0z7UTJ4",
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "Explosive Harness",
+            "sort": 400000,
+            "system": {
+                "actionCategory": {
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>The rat is connected to an explosive harness, which displays an ominous 1-minute pocket watch. The harness will explode after 1 minute (10 rounds) if it is not disabled, or immediately if the rat takes fire damage. This explosion deals [[/r {2d6}[fire]]]{2d6 fire damage} to the rat and all creatures within a @Template[type:burst|distance:5]{5-foot burst} (@Check[type:reflex|dc:15|basic:true] save) and ignites flammable objects in the area.<br />If the rat is @Compendium[pf2e.conditionitems.Grabbed]{Grappled}, @Compendium[pf2e.conditionitems.Restrained]{Restrained}, or @Compendium[pf2e.conditionitems.Unconscious]{Unconscious}, an adjacent creature can attempt to safely remove and disable the harness with a successful @Check[type:nature|dc:17] or @Check[type:thievery|dc:17] check or a successful @Check[type:tanning-lore|dc:15] check. An attempt to disable the harness uses two actions.</p>"
```

Comment:
What the fuck??

Reply:
Evidently Paizo thinks there is more than one way to skin a rat. Although Tanning Lore is a bit on the nose, I admit